config/cesm/machines/config_machines.xml

```diff
@@ -116,6 +116,57 @@ This allows using a different mpirun command to launch unit tests
     </environment_variables>
   </machine>
 
+  <machine MACH="arc4">
```

Comment:
You have a second copy of this at line 2802, please remove one.   This file is generally sorted alphabetically by MACH value.